Kconfig.thingset

```diff
@@ -106,6 +106,12 @@ config THINGSET_METADATA_ENDPOINT
 	  The endpoint is disabled by default in order to reduce memory footprint for constrained
 	  devices.
 
+config THINGSET_PROGRESSIVE_IMPORT_EXPORT
```

Comment:
I think we should remove the Kconfig in order to make the code easier to read.

The functions won't increase code size if they are not used, so we don't really need the Kconfig.